packages/@coorpacademy-components/src/molecule/skill-picker-modal/index.js

```diff
@@ -128,6 +129,7 @@ const SkillPickerModal = (props, context) => {
         name: 'bullseye-arrow',
         backgroundColor: '#DDD1FF'
       }}
+      detectScrollbar={detectScrollbar}
```

Comment:
```suggestion
      detectScrollbar={true}
```
pas besoin de créer une variable tu ne l'utilise pas ailleurs

Reply:
eslint: Value must be omitted for boolean attributes